src/holochainClient.js

```diff
@@ -111,9 +112,10 @@ export function conductorInstanceIdbyDnaAlias (instanceId) {
 }
 
 let holochainClient
+let hcFlag = false
```

Comment:
can we call this something more descriptive?

`isInitiatingHcConnection` or similar